src/app/ipv/middleware.js

```diff
@@ -406,21 +423,13 @@ async function handleJourneyPage(req, res, next, pageErrorState = undefined) {
 }
 
 async function handleJourneyAction(req, res, next) {
-  const currentPageId = req.params.pageId;
-  const pagesNotUsingOAuthSessionId = [
-    PAGES.NO_PHOTO_ID_EXIT_FIND_ANOTHER_WAY,
-    PAGES.NO_PHOTO_ID_SECURITY_QUESTIONS_FIND_ANOTHER_WAY,
-    PAGES.PAGE_MULTIPLE_DOC_CHECK,
-    PAGES.PYI_CRI_ESCAPE,
-    PAGES.PYI_SUGGEST_OTHER_OPTIONS,
-  ];
+  const pageId = req.params.pageId;
 
   try {
-    if (pagesNotUsingOAuthSessionId.includes(currentPageId)) {
-      checkForIpvSessionId(req, res);
-    } else {
-      checkForIpvAndOauthSessionId(req, res);
```

Comment:
Is it safe to drop the check for the OAuth session ID?

Reply:
I believe the only scenario we have a `clientOauthSessionId` is for the page `pyi-timeout-recoverable` (the mobile cross-browser scenario). I moved the check to `validateSessionAndPage` function which checks specifically for that page